bls/src/multi_sig_slow.rs

```diff
@@ -73,26 +81,40 @@ impl MultiSignature {
     // the aggregator simply adds each signer's output. In that model, signer does more work but in the
     // implemented model, aggregator does more work and the same signer implementation can be used by
     // signers of "slow" and "fast" implementation.
-    pub fn new(sigs_and_ver_keys: &[(Signature, VerKey)]) -> Signature {
+    pub fn new<'a, T>(sigs_and_ver_keys: T) -> Signature
+    where
+        T: IntoIterator<Item = &'a (Signature, VerKey)>,
```

Comment:
I think this should probably be a tuple of references instead of a reference to a tuple, so it would be ``(&'a Signature, &'a Verkey)